### DISCUSSION TITLE
Como vocês conciliam/organizam o tempo entre trabalho fixo, freelas e projetos pessoais?

### DISCUSSION THREAD

**Discussion by kvnol** [frontendbr/forum#1625] [Discussão]
O dilema de um garoto de 22 anos que quer fazer tudo, mas não tem (ou apenas não consegue organizar) tempo pra nada.

**Comment by vsalbuq** [comment]
Regra de pareto: Foque nos 20% que te dão 80% do retorno. Tempo = priorização + organização.

Primeiro você precisa priorizar o que é importante. Priorizar é prioridade. A maioria das pessoas está cheia de técnicas de organização, mas não evoluem por não terem foco. Só com isso, você vai ganhar tempo, pois vai cortar aquilo que não te beneficia em nada.

Para priorizar, você precisa ter uma escala de valores bem definida. Para isso, você precisa saber onde quer chegar e definir como vai chegar lá. 

Para saber onde quer chegar, faça o exercício: Se de 3 a 5 anos à frente tudo na minha vida tivesse dado certo, como seria meu dia? Em qual empresa está trabalhando? É home office? Quais as principais características dos seus colegas de trabalho? O que você faz a maior parte do dia? Quanto ganha?

Depois de saber onde quer chegar (definir o dia perfeito), você precisa enxergar cada tarefa como um caminho e se perguntar "Esse caminho me leva para mais perto de onde quero chegar ou para mais longe do meu dia perfeito?" Com isso você consegue ter a clareza e focar naquilo que realmente é importante.

Não aceite freelas que não vão te acrescentar naquilo que você mais valoriza. Defina um valor mínimo de hora por seu tempo, defina características básicas que o freela deve ter e não negocie em relação a essas regras.

Não inicie projetos pessoais que não te levarão para mais próximo do seu objetivo.

Não permaneça em empregos que não estão alinhados com o seu objetivo.

Se depois disso, você ainda estiver com dificuldade de encontrar tempo para fazer as coisas (o que é bem difícil acontecer, e eu já dei esse conselho para várias pessoas), você pode focar em técnicas de administração de tempo.

Administração de tempo se divide em duas fases: uma operacional e uma gerencial. Na fase gerencial você define tudo o que tem que fazer. É o seu planejamento diário, semanal e mensal, e a visão de como isso te leva ao seu objetivo final.

Na fase operacional, você simplesmente vira a chave e executa tudo o que planejou, sem questionar nada. O momento de criticar as tarefas é no planejamento.

Com o tempo você vai adquirindo prática em planejar e executar, e vai ver tarefas e planos que não fazem sentido.

Eu duvido que você não alcance tudo o que quiser se fizer essas coisas. Demora um pouco, mas vale muito a pena. Você ainda é novo.

**Comment by rodrigoms2004** [comment]
Sensacional vou salvar esse post
